### DISCUSSION TITLE
[Feature] ML Server

### DISCUSSION THREAD

**Discussion by dabucz** [immich-app/immich#10155] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great if it was possible to use ollama server as a machine learning server.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
What does this mean? ollama is a server for large language models, which Immich doesn't use.

**Comment by dabucz** [comment]
I thought you could use it for the face detection because for some usecases its easier rather than using the ml docker container.

**Comment by zackpollard** [comment]
This doesn't make sense in the context of Immich, if we were to use any LLM's, we'd integrate them into our existing ML stack.

**Comment by dabucz** [comment]
oh okay
